tests/unit/cmd/sca/test_scan.py

```diff
@@ -625,6 +630,9 @@ def test_sca_scan_subdir_with_ignored_vuln(
             ],
         )
 
+        ignored_vulns = sca_scan_params_mock.call_args.kwargs["ignored_vulnerabilities"]
+        assert [i.identifier for i in ignored_vulns] == ["GHSA-rrm6-wvj7-cwh2"]
```

Comment:
don't you want to try adding CVE? I don't see how this change is related to the handling of CVEs

Reply:
This change is not directly related to the handling of CVEs, I changed this to put it in a separate commit.
This test needed improvement since the cassette made it a false positive, as the Gitguardian API response will never have any incident. I added this to test we at least send the correct option to ignore vulnerabilities.